docs/docs/examples/mini-examples/prometheus-alerting.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Prometheus alerting
+description: Different methods for monitoring and observability with Prometheus monitoring.
+last_update:
+  author: Dennis Hume
+sidebar_custom_props:
+  logo: images/dagster-primary-mark.svg
+---
+
+In this example, we'll explore different approaches to implementing [Prometheus](https://prometheus.io/) monitoring in Dagster. Prometheus is a powerful monitoring system that collects metrics from your applications and provides alerting capabilities. When integrated with Dagster, it enables comprehensive observability for your data pipelines, allowing you to track asset execution times, failure rates, and custom business metrics.
+
+## Problem: Limited pipeline observability
+
+Without proper monitoring, data pipeline issues can go unnoticed until they cause downstream problems. Common challenges include:
+
+- **Silent failures**: Assets fail but no one is notified
+- **Performance degradation**: Slow assets aren't detected early
+- **Lack of business metrics**: No visibility into data quality or business KPIs
+- **Manual monitoring**: Checking pipeline health requires manual dashboard reviews
+
+Prometheus monitoring addresses these issues by:
+
+- Automatically collecting and storing metrics
+- Providing alerting based on configurable thresholds
+- Enabling historical analysis of pipeline performance
+- Supporting custom business metrics alongside technical metrics
+
+### Solution 1: Prometheus resource setup
+
+The first step is configuring a Prometheus resource that connects to your Prometheus push gateway. This provides the foundation for all monitoring functionality.
+
+<CodeExample
+  path="docs_projects/project_mini/src/project_mini/defs/prometheus_alerting/resources.py"
+  language="python"
+  title="src/project_mini/defs/prometheus_alerting/resources.py"
+  startAfter="start_prometheus_resources"
+  endBefore="end_prometheus_resources"
+/>
+
+Key configuration parameters:
+
+- **`gateway`**: URL of your Prometheus push gateway
+- **`job_name`**: Identifier for your Dagster instance
+- **`scrape_interval`**: How often Prometheus collects metrics
+- **`scrape_timeout`**: Maximum time to wait for metric collection
+- **`evaluation_interval`**: How often alerting rules are evaluated
+
+### Solution 2: Asset monitoring
+
+Once the resource is configured, you can create assets that push basic metrics to Prometheus using the built-in functionality.
+
+<CodeExample
+  path="docs_projects/project_mini/src/project_mini/defs/prometheus_alerting/prometheus_assets.py"
+  language="python"
+  title="src/project_mini/defs/prometheus_alerting/prometheus_assets.py"
+  startAfter="start_basic_prometheus_asset"
+  endBefore="end_basic_prometheus_asset"
+/>
+
+This basic approach automatically sends default metrics to Prometheus, including:
+
+- Asset execution start and completion times
+- Asset success/failure status
+- Job and asset metadata
+
+| Metric Type             | Automatically Collected | Custom Configuration |
```

Comment:
I would delete this table, since it mostly duplicates the info in the list above, and the custom metrics info is covered in the section below.